packages/devtools_app/lib/src/utils.dart

```diff
@@ -404,6 +404,10 @@ class JsonUtils {
   static bool hasJsonData(String data) {
     return data != null && data.isNotEmpty && data != 'null';
   }
+
+  /// Pretty print a JSON payload.
+  static String prettyPrint(Map<String, Object> json) =>
```

Comment:
this might be a nice extension method on `Map<String, Object>`

Reply:
Good idea, done.